### PR TITLE
add pending resolution to filters

### LINKED ISSUE #1549: Add Pending Resolution filter.
closed questions, whose scheduled resolution time is in the past, sorted by most recent scheduled resolution

### PR DESCRIPTION
**Pull request by lsabor**
closes #1549

### CODE REVIEW

File: front_end/src/app/(main)/questions/helpers/filters.ts [hunk 1]
```diff
@@ -51,13 +51,14 @@ const POST_TYPE_LABEL_MAP: Record<ForecastType, string> = {
...
   [PostStatus.OPEN]: "Open",
-  [PostStatus.UPCOMING]: "Upcoming",
-  [PostStatus.RESOLVED]: "Resolved",
   [PostStatus.CLOSED]: "Closed",
-  [PostStatus.PENDING]: "In Review",
-  [PostStatus.DRAFT]: "Draft",
+  [PostStatus.PENDING_RESOLUTION]: "Pending Resolution",
+  [PostStatus.RESOLVED]: "Resolved",
   [PostStatus.DELETED]: "Deleted",
   [PostStatus.REJECTED]: "Rejected Posts",
```
Comment (lsabor) [inline front_end/src/app/(main)/questions/helpers/filters.ts:63]: reordered these options and added `PENDING_RESOLUTION`

**Comment by ncarazon** [comment]
@lsabor It won't be an easy task and require a refactor of `PopoverFilter` component to allow pop-up elements to update multiple URL params at once. I can take a look once I finish with all my high-pri tasks, but ideally, we would move that into a separate ticket, so it wouldn't block this PR.

**Review by hlbmtc — APPROVED** [review]
LGTM!

**Comment by lsabor** [comment]
@ncarazon Thanks, let's look at it on another ticket!

**Comment by lsabor** [comment]
Created the issue @ncarazon https://github.com/Metaculus/metaculus/issues/1574

**Pull request merged**